docs/src/content/faq.mdx

```diff
@@ -48,27 +81,280 @@ After receiving the NFT, follow these steps to set up the Icebreaker platform:
 
 For detailed instructions on each step, follow the links above.
 
-## How do I get access to the private channel with Icebreaker documentation?
+### What endpoints do I need to open in my firewall for the Icebreaker node?
 
-After receiving your Icebreaker NFT:
+To function properly, your Icebreaker node requires the following ports to be open in your firewall:
 
-- Notify the Blockfrost team that you've received your NFT
-- A team member will add you to the private Icebreakers channel
-- Access the comprehensive documentation in the announcement channel
-- Follow the setup instructions provided there.
+- `3000/tcp` (or custom port): For incoming API requests from Blockfrost
+- `8081/tcp`: Prometheus metrics endpoint for monitoring (optional but recommended). Metrics can be disabled altogether if you prefer.
 
-If you've received your NFT but don't have channel access yet, post a message on [Discord](https://discord.com/invite/inputoutput) in the `#ask-blockfrost` channel for assistance.
+Ensure these ports allow incoming connections from the Blockfrost server IPs.
 
-## Do I need confirmation after receiving the NFT to proceed?
+**Example for Ubuntu with ufw**
 
-No, you don't need additional confirmation after receiving the NFT.
-Once the NFT is in your wallet, you can proceed with installation and setup.
-However, you'll need access to the private Icebreakers channel for complete documentation, which is granted after notifying the team that you've received your NFT.
+You can verify port accessibility using:
+```bash
+sudo ufw allow 3000/tcp
+sudo ufw allow 8081/tcp
+```
 
-## Can I transfer my Icebreaker NFT to another wallet address?
+**Example using a computer outside of the server network**
+
+You can verify port accessibility from the external world by using another computer outside of the server network: 
+
+```bash
+telnet example.com 3000
```

Comment:
Shouldn’t this also say `YOUR_PUBLIC_IP` instead of `example.com`?

Corrected in:
* #290